package.json

```diff
@@ -14,7 +14,7 @@
     "url": "https://github.com/twilio/twilio-node.git"
   },
   "dependencies": {
-    "request": "2.55.x",
+    "request": "2.70.x",
```

Comment:
Minor detail, but I think you can update to `^2.72.x` as https://github.com/twilio/twilio-node/pull/167/files